rma_product_exchange/models/procurement_group.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Akretion
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from odoo import api, models
+
+
+class ProcurementGroup(models.Model):
+    _inherit = "procurement.group"
+
+    @api.model
+    def run(self, procurements, raise_user_error=True):
+        if self.env.context.get("rma_item"):
```

Comment:
Is this stuff really usefull ?
You already override `rma_make_picking.wizard._get_product()` it is enough and you don't need to do this stuff with context, or maybe I am missing something

Reply:
@florian-dacosta , it's removed.